packages/textlint-server/package-lock.json

```diff
@@ -1,363 +0,0 @@
-{
-  "name": "vscode-textlint-server",
```

Comment:
https://github.com/taichi/vscode-textlint/blob/7f73c80084c8c80b91ec59612746ac717ae41785/package.json#L4-L7
This repo use workspaces. This file is not needed.